templates/posts_report.twig

```diff
@@ -1,33 +1,41 @@
 {% block content %}
 
-    <h1>Posts Report</h1>
+    <h1>{{ __( 'Posts Report', 'planet4-master-theme-backend' ) }}</h1>
     <div class="tablenav top">
         <div class="alignleft actions">
-            <label>Filters: </label>
-            <label for="from">Date From</label>
-            <input type="text" id="from" name="from">
-            <label for="to">to</label>
-            <input type="text" id="to" name="to">
+            <label>{{ __( 'Filters:', 'planet4-master-theme-backend' ) }} </label>
+            <label for="from">{{ __( 'Date From', 'planet4-master-theme-backend' ) }}</label>
+            <input type="text" id="from" name="from" autocomplete="off">
```

Comment:
Added a autocomplete off here, the datepicker having browser text field autosuggestion, looks me little annoying.

Reply:
Would it make sense to drop the date picker and directly use an `<input type="date" />` ? (I don't know what browser our users are using https://caniuse.com/#feat=input-datetime )

Reply:
@comzeradd - Any suggestions on above?

Reply:
Since this is for Editors, I'd say it's safe to use the native `type="date"` that's supported by all 3 major browsers.